lib/macho/macho_file.rb

```diff
@@ -362,13 +378,25 @@ def rpaths
     #  file.change_rpath("/usr/lib", "/usr/local/lib")
     # @param old_path [String] the old runtime path
     # @param new_path [String] the new runtime path
+    # @param options [Hash]
+    # @option options [Boolean] :strict (true) whether or not to fail loudly
+    #  with an exception if the change cannot be performed
+    # @return [void]
     # @return [void]
     # @raise [MachO::RpathUnknownError] if no such old runtime path exists
     # @raise [MachO::RpathExistsError] if the new runtime path already exists
-    def change_rpath(old_path, new_path)
+    def change_rpath(old_path, new_path, options = {})
       old_lc = command(:LC_RPATH).find { |r| r.path.to_s == old_path }
-      raise RpathUnknownError.new(old_path) if old_lc.nil?
-      raise RpathExistsError.new(new_path) if rpaths.include?(new_path)
+
+      if rpaths.include?(new_path)
+        return unless options.fetch(:strict, true)
+        raise RpathExistsError.new(new_path)
+      end
+
+      if old_lc.nil?
+        return unless options.fetch(:strict, true)
+        raise RpathUnknownError.new(old_path)
+      end
```

Comment:
This change needs to be reverted. The modification methods in `MachOFile` need to always raise an exception, independently of `options` (because `options` currently only matters if multiple Mach-O slices are involved).